### ISSUE TITLE
.css file not added into distribution in JS only library 

### ISSUE THREAD

**Issue by cagline** [rollup/rollup#3736]
I'm writing JS library using [rollup-starter-lib](https://github.com/rollup/rollup-starter-lib) as base of my JS library

It has `main.js` file & `main.css` file, There is no `html` file.
`main.js` file is all about DOM manipulation, `main.css` file is necessary in implementation steps, to easily integrate this JS library, rather giving an instruction to user.  

_I feel like it's .css file not creating due to tree shaking because of css not used in this JS library itself._

Current Distribution
```
-dist
--my-faceapi-js-lib.cjs.js
--my-faceapi-js-lib.esm.js
--my-faceapi-js-lib.umd.js
```
Expected Distribution
```
-dist
--my-faceapi-js-lib.cjs.js
--my-faceapi-js-lib.esm.js
--my-faceapi-js-lib.umd.js
--my-faceapi-js-lib.css
```
copy `main.css` file into `dist` folder not expecting since it's not minifying. 

rollup.config.js
```
import resolve from '@rollup/plugin-node-resolve';
import commonjs from '@rollup/plugin-commonjs';
import pkg from './package.json';
import path from 'path'
import postcss from 'rollup-plugin-postcss'

export default [
	{
		input: 'src/main.js',
		output: {
			name: 'howLongUntilLunch',
			file: pkg.browser,
			format: 'umd'
		},
		plugins: [
                         postcss({
				extract: path.resolve('dist/my-faceapi-js-lib.css'),
			}),
			resolve(), // so Rollup can find `ms`
			commonjs() // so Rollup can convert `ms` to an ES module
		]
	},
	{
		input: 'src/main.js',
		external: ['ms'],
		output: [
			{ file: pkg.main, format: 'cjs' },
			{ file: pkg.module, format: 'es' }
		]
	}
];
```
main.js

```
import * as faceapi from "face-api.js";
...
import 'main.css';
...
...
```
Basically there is not usage of `main.css` within JS library it self. but it's necessary when integrate this library.

**Comment by vladshcherbin** [comment]
@cagline try changing css import path: `import './main.css`

**Comment by shellscape** [comment]
_If you are sure this is a bug or documentation issue, please take the time to refile this issue **using the issue template**._

Hey there, thanks for opening an issue - Unfortunately you missed (or may have disregarded) the message about issue templates being _required_. Because our issue template was removed, we assume that this is a support request, and that's not something we offer here. Also, if you refile, please don't ignore Issue sections. They're in the template because they're useful to the maintainers of this project.

**Comment by cagline** [comment]
BTW, After that i update rollup version to `2.26.3`  from `1.29.0` now it's generate only following two files along with below error.  It's seams there is an issue with rollup or postCss plugin.

> [!] Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)
> src/main.css (1:0)

```
--my-faceapi-js-lib.umd.js
--my-faceapi-js-lib.css
```

**Comment by cagline** [comment]
> @cagline try changing css import path: `import './main.css'`
already tried out

**Comment by cagline** [comment]
Hi @shellscape , Thanks for the reply.
I just open an issue using the issue template.

https://github.com/rollup/rollup/issues/3737
